<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetVectorSpinBox</class>
 <widget class="QWidget" name="WidgetVectorSpinBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>196</width>
    <height>20</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="currentIndex">
      <number>4</number>
     </property>
     <widget class="QWidget" name="intPage">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSpinBox" name="intSpinBoxX">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="suffix">
          <string/>
         </property>
         <property name="prefix">
          <string>X: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="intSpinBoxY">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="prefix">
          <string>Y: </string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="doublePage">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBoxX">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="prefix">
          <string>X: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBoxY">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="prefix">
          <string>Y: </string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="intPage3d">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSpinBox" name="intSpinBox3dX">
         <property name="prefix">
          <string>X: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="intSpinBox3dY">
         <property name="prefix">
          <string>Y: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="intSpinBox3dZ">
         <property name="prefix">
          <string>Z: </string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="doublePage3d">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox3dX">
         <property name="prefix">
          <string>X: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox3dY">
         <property name="prefix">
          <string>Y: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox3dZ">
         <property name="prefix">
          <string>Z: </string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="intPage4d">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSpinBox" name="intSpinBox4dX"/>
       </item>
       <item>
        <widget class="QSpinBox" name="intSpinBox4dY"/>
       </item>
       <item>
        <widget class="QSpinBox" name="intSpinBox4dZ"/>
       </item>
       <item>
        <widget class="QSpinBox" name="intSpinBox4dW"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="doublePage4d">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox4dX"/>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox4dY"/>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox4dZ"/>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox4dW"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>doubleSpinBox4dX</tabstop>
  <tabstop>doubleSpinBox4dY</tabstop>
  <tabstop>doubleSpinBox4dZ</tabstop>
  <tabstop>doubleSpinBox4dW</tabstop>
  <tabstop>intSpinBoxX</tabstop>
  <tabstop>intSpinBoxY</tabstop>
  <tabstop>doubleSpinBoxX</tabstop>
  <tabstop>doubleSpinBoxY</tabstop>
  <tabstop>intSpinBox3dX</tabstop>
  <tabstop>intSpinBox3dY</tabstop>
  <tabstop>intSpinBox3dZ</tabstop>
  <tabstop>doubleSpinBox3dX</tabstop>
  <tabstop>doubleSpinBox3dY</tabstop>
  <tabstop>doubleSpinBox3dZ</tabstop>
  <tabstop>intSpinBox4dX</tabstop>
  <tabstop>intSpinBox4dY</tabstop>
  <tabstop>intSpinBox4dZ</tabstop>
  <tabstop>intSpinBox4dW</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
